pymc3/distributions/mixture.py

```diff
@@ -197,21 +212,21 @@ class NormalMixture(Mixture):
         the component standard deviations
     tau : array of floats
         the component precisions
+    distshape : shape of the Normal component
```

Comment:
it looks like this is already called `dist_shape` in `generate_samples` (instead of `distshape`)... is it easy to change that here as well?

Reply:
I will change it to `comp_shape` to avoid confusion.